web/source/kmwbase.ts

```diff
@@ -73,17 +87,19 @@ namespace com.keyman {
     touchAliasing: dom.DOMEventHandlers;
 
     // Defines option-tracking object as a string map.
-    options: { [name: string]: string; } = {
-      'root':'',
-      'resources':'',
-      'keyboards':'',
-      'fonts':'',
-      'attachType':'',
-      'ui':null,
-      'setActiveOnRegister':'true'
+    options: OptionType = {
+      root: '',
+      resources: '',
+      keyboards: '',
+      fonts: '',
+      attachType: '',
+      ui: null,
+      setActiveOnRegister: 'true', // TODO: convert to boolean
+
+      // Determines whether or not KeymanWeb should display its own alert messages
+      useAlerts: true
     };
 
-
     // Stub functions (defined later in code only if required)
     setDefaultDeviceOptions(opt){}
```

Comment:
Also consumes the new `OptionType` type; will also have related changes for the implementations in kmwnative.ts and kmwembedded.ts.